### PR TITLE
chore(create-medusa-app): Remove Admin + Gatsby starter from npx

### PR DESCRIPTION
**Pull request by olivermrbl**
What
Remove Medusa Admin and Gatsby starters from `npx`. 

The former is removed, as we now ship admin as a plugin. The latter are removed because they are no longer maintained.

### CODE REVIEW

File: packages/create-medusa-app/src/index.ts [hunk 1]
```diff
@@ -126,23 +116,13 @@ export const run = async (): Promise<void> => {
...
     Medusa API
     cd ${projectRoot}/backend
     yarn start
-
-    Admin
-    cd ${projectRoot}/admin
-    yarn start
   `)
 
   if (hasStorefront) {
```
Comment (kasperkristensen) [inline packages/create-medusa-app/src/index.ts:128]: we should change what we print if the Next starter is included:

```
 Storefront
 cd ${projectRoot}/storefront
 yarn dev
```
Running `yarn start` will throw an error as that requires that you build the starter before running the command.

Comment (olivermrbl) [reply packages/create-medusa-app/src/index.ts:128]: Got it, will include in this PR 👍

**Review by kasperkristensen — APPROVED** [review]

**Review by srindom — APPROVED** [review]
LGTM - is medusa-starter-default updated to use admin package?

**Comment by olivermrbl** [comment]
> is medusa-starter-default updated to use admin package?

Not yet, but for v1.8.0 it will be 👍

**Pull request merged**